proposals/2021-02-enable-ibc-transfers/proposal.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Enable IBC transfers",
+  "description": "The Proposal enables transfering and recieving assets using the IC20 protocol on the Cosmos Hub. If this proposal passes, there will be IBC assets available in the Bank module of the Hub and ATOM will be available on Zones connected over IBC.\n Iqlusion believes that the IBC software is suffciently stable for small amounts of value transfer. We expect there to be issues with stuck funds and UX confusion but overcoming these issues will only happen once IBC is live.\n Security Model\n Tendermint full nodes produce agreement under the assumption that at most ⅓ of the voting power held by validators is Byzantine.\n IBC\nIBC is a protocol for authenticated message passing between heterogeneous sovereign blockchains. IBC requires trusting that chains on both sides of the connections operate within their security model.\n Incentive Security Extensions\n IBC has a facility to support freezing connections once a violation of the security model has occurred. The set of criteria for detecting such attacks continues to evolve and is a constant focus of research.",
+  "changes": [
+    {
+      "subspace": "transfer",
+      "key":"send_enabled",
+      "value": true
+    },
+    {
+      "subspace": "transfer",
+      "key":"recieve_enabled",
+      "value": true
+    }
+  ],
+  "deposit": [
+    {
+      "denom": "uatom",
+      "amount": "1000000"
+    }
+  ]
```

Comment:
This returned:
```
Error: json: cannot unmarshal array into Go value of type string
```

Not sure if this got changed or if there is a syntax error, but this is what I used for testing on gaia v4.1.0:
```
  "deposit": "1000000uatom"
}
```